benchmarks/src/helpers/paths.jl

```diff
@@ -0,0 +1,50 @@
+# Collection of path function mimicking DrWatson's functions
+using DrWatson
+@quickactivate :benchmarks
+
+export resultsdir, tablesdir, helpersdir, aggdatadir, postprocessdir
+export get_configname, get_config, get_simfile_name, get_simfile, get_statsfile_name, get_statsfile, get_benchfile_name, get_benchfile
+export get_config_size, get_meta_config_info
+
+helpersdir(args...) = srcdir("helpers", args...)
+
+resultsdir(sim_name, args...) = datadir("sims", sim_name, args...)
+
+tablesdir(sim_name, slurm_id, args...) = datadir("exp_pro", sim_name, slurm_id, args...)
+aggdatadir(sim_name, slurm_id, args...) = tablesdir(sim_name, slurm_id, "autogen", args...)
+postprocessdir(args...) = scriptsdir("post_processing", args...)
+
+function get_configname(sim_name, arch)
```

Comment:
all these get_ prefixes are for Java programmers. we can just use `configname`

Reply:
Fair enough, it'd be less naming noise.